packages/ui/src/domainModules/collectionMammals/components/MammalManager/ResultOptionsBar/index.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Button, Grid, Icon, Menu, Popup, Segment } from 'semantic-ui-react'
+import 'common/dist/semantic.css' // eslint-disable-line
+import 'react-rangeslider/lib/index.css'
+
+const propTypes = {
+  activeItem: PropTypes.string.isRequired,
+  onExportCSV: PropTypes.func.isRequired,
+  onFormTabClick: PropTypes.func.isRequired,
+  onSettingClick: PropTypes.func.isRequired,
+  onTableTabClick: PropTypes.func.isRequired,
+  tabContent: PropTypes.string.isRequired,
+}
+
+export class ResultOptionsBar extends Component {
+  render() {
+    const {
+      activeItem,
+      onExportCSV: handleExportToCSV,
+      onFormTabClick: handleFormTabClick,
+      onSettingClick: handleSettingClick,
+      onTableTabClick: handleTableTabClick,
+      tabContent,
+    } = this.props
+
+    return (
+      <div>
+        <Menu attached="top" tabular>
+          <Menu.Item
+            active={activeItem === 'form'}
+            name="form"
+            onClick={() => handleFormTabClick()}
```

Comment:
in this case we should pass through the event, so `(event) => handleFormTabClick(event)` so that the MammalManager can decide what to do with it. please do that for all onClick in this component.

(I still think it should work with just `onClick={handleFormTabClick}`... but maybe it is some special case with the storybook...)